### DISCUSSION TITLE
Exercism vs FreeCodeCamp

### DISCUSSION THREAD

**Discussion by audreyhki** [USU-ITLS-x265-23-summer/class-repo#82] [Week 2 - Discussions]
Well, the goal I set for myself, to not fall behind in class the summer semester has slightly escaped me, but I chasing back after it with a vengeance. The song of the summer is coming along fantastically. 
Exercism has caused me some amount of grief. I'm unsure if there is a tutorial or instruction base I missed or am lacking but I find the project instructions more like a riddle than a guide. I assume this is on purpose to instigate a problem-solving initiative from the participant, but I find it frustrating and demotivating to work on something I have no background on and am given no clear instructions.
FreeCodeCamp is much easier to navigate and learn from. I found it similar to the GitHub tutorials which were helpful.  Being modeled a concept, then repeating it for yourself is my preferred learning method. 
When I would feel stuck on Exercism I would turn to Google and YouTube to look for a solution. When I feel stuck on FreeCodeCamp I check my work, re-read the instructions, and can figure it out on my own.

**Comment by metasean** [comment]
>  I assume this is on purpose to instigate a problem-solving initiative from the participant

Fwiw: many of the concepts that you were introduced to in Exercism you'll practice in FreeCodeCamp.  Also, the Exercism exercises are only included in week 2.
